examples/tutorials/alpha-zero-tic-tac-toe/tictactoe/TicTacToePlayers.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+
+"""
+Random and Human-ineracting players for the game of TicTacToe.
+
+Author: Evgeny Tyurin, github.com/evg-tyurin
+Date: Jan 5, 2018.
+
+Based on the OthelloPlayers by Surag Nair.
+
+"""
+
+
+class RandomPlayer:
+    def __init__(self, game):
+        self.game = game
+
+    def play(self, board):
+        a = np.random.randint(self.game.getActionSize())
+        valids = self.game.getValidMoves(board, 1)
+        while valids[a] != 1:
+            a = np.random.randint(self.game.getActionSize())
+        return a
+
+
+class HumanTicTacToePlayer:
+    def __init__(self, game):
+        self.game = game
+
+    def play(self, board):
+        # display(board)
+        valid = self.game.getValidMoves(board, 1)
+        for i in range(len(valid)):
+            if valid[i]:
+                print(int(i / self.game.n), int(i % self.game.n))
+        while True:
+            # Python 3.x
+            a = input()
+            # Python 2.x
+            # a = raw_input()
```

Comment:
nit: I think determined is all python >3 so it's probably safe to remove this

Reply:
Good point. Those comments from the imported code have been removed.